### PR TITLE
View the logs from deployments

### PR DESCRIPTION
**Pull request by 0xmichalis**
Origin PRs: https://github.com/openshift/origin/pull/3943 + https://github.com/openshift/origin/pull/5393

### CODE REVIEW

**Comment by kargakis** [comment]
@adellape this can be merged now

**Comment by kargakis** [comment]
Updated to include older deployment logs which depends on https://github.com/openshift/origin/pull/5393

**Comment by kargakis** [comment]
@adellape ping

File: dev_guide/deployments.adoc [hunk 1]
```diff
@@ -210,6 +210,28 @@ To see what the rollback would look like without performing the rollback:
...
+Note that if the deployment is successful, there will be no logs to view. Logs 
+can be retrieved either when the deployment is running or fails. There is also
+the ability to view logs from older deployments:
+
+----
+$ oc logs --version=1 dc/<deployment_config> 
+----
+
+The above command will return the logs from the first deployment of the provided
+deployment config, iff that deployment exists (has failed and hasn't been manually
```
Comment (adellape) [inline dev_guide/deployments.adoc]: s/iff/if/

**Comment by adellape** [comment]
@kargakis Minor issue inline, otherwise LGTM, thanks!

**Comment by kargakis** [comment]
I wanted to express "if and only if". Isn't this idiom valid for this doc?

https://en.wikipedia.org/wiki/If_and_only_if

**Comment by adellape** [comment]
> I wanted to express "if and only if". Isn't this idiom valid for this doc?

@kargakis Thanks, I was not familiar. I think for the doc then we should just spell it out.

File: dev_guide/deployments.adoc (hunk 1 above)
Comment (kargakis) [inline dev_guide/deployments.adoc]: updated

**Pull request merged**